docs/synthetics/ci-cd-integration.md

```diff
@@ -0,0 +1,300 @@
+                  --header 'content-type: application/json' \
+                    --data '[{"monitorId": 276, "url":'$target_url'}]' > results.txt
+          cat results.txt
+          if [ $(head -1 results.txt | grep -c 'failed') -ne 0 ]; then exit 1; fi
+```
+
+**GitHub Actions Logs**
+
+On every deployment event, the action will be invoked and the action logs will contain the results.
+
+<img
+  class="content-modal-image"
+  alt="CI/CD GitHub Secret"
+  src="../../images/synthetics/ci-cd-github-actions-log.png"
+  title="Add Secret in Github"
+/>
+
+
+### Bitbucket Pipelines
+
+**Create API Token Secret**
+
+Create a secret from the Repository Settings -> Repository Variables, for Sematext API Token.
+
+<img
+  class="content-modal-image"
+  alt="CI/CD BitBucket Secret"
+  src="../../images/synthetics/ci-cd-bitbucket-secret.png"
+  title="Add Secret in BitBucket"
+/>
+
+**Add Run Monitor Job to Bitbucket Pipeline Workflow YAML**
+
+Add Run Monitor step to `bitbucket-pipelines.yml`. Add the below steps after the deploy step in your pipeline configuration.
+
+```yaml
+image: node:10.15.3
+
+pipelines:
+  custom:
+    sematext:
+      - step:
+          script:
+            - curl -H "authorization:apiKey $SEMATEXT_API_TOKEN" -H "accept:text/plain" -H "content-type:application/json" -s -X POST -d "[{\"monitorId\":276}]" https://apps.sematext.com/synthetics-api/api/v3/apps/12345/monitors/runs > results.txt
+            - cat results.txt
+            - if [ $(head -1 results.txt | grep -c 'failed') -ne 0 ]; then exit 1; fi
+
+```
+
+### GitLab CI/CD
+
+**Create API Token Variable**
+
+Create a CI/CD variable from the Settings -> CI/CD -> Variables, for Sematext API Token.
+
+<img
+  class="content-modal-image"
+  alt="CI/CD GitLab Secret"
+  src="../../images/synthetics/ci-cd-gitlab-secret.png"
+  title="Add Secret in GitLab"
+/>
+
+**Configure Run Monitor Step in Pipeline YAML**
+
+Configure run monitor stage in `.gitlab-ci.yml` to trigger run after deployment and upload the artifacts.
+
+```yaml
+stages:
+  - deploy
+deploy-app:
+    stage: deploy
+    image: docker:stable
+    script:
+      - echo "script to deploy your App"
+run-sematext-monitor:
+    stage: .post
+    image: docker:stable
+    before_script:
+    - apk add --update curl && rm -rf /var/cache/apk/*
+    script:
+        - curl -H "authorization:apiKey $SEMATEXT_API_TOKEN" -H "accept:text/plain" -H "content-type:application/json" -s -X POST -d "[{\"monitorId\":276}]" https://apps.sematext.com/synthetics-api/api/v3/apps/12345/monitors/runs > results.txt
+        - cat results.txt
+        - if [ $(head -1 results.txt | grep -c 'failed') -ne 0 ]; then exit 1; fi
+    artifacts:
+        expose_as: 'Sematext Monitor Results'
+        paths: ['results.txt']
+```
+
+### Vercel
+
+Configure [GitHub Action](#gitlab-cicd) to run the monitor on deployment event. If the Vercel project repository is GitHub, the action will be automatically triggered on PR or production deployment.
```

Comment:
GitHub or GitLab?  I see both mentioned in this line.

Reply:
Should be `#github-actions`. Will update